picard/ui/options/metadata.py

```diff
@@ -261,4 +263,106 @@ def save_changes(self):
         self.accept()
 
 
+class ScriptExceptionSelector(PicardDialog):
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.ui = Ui_ExceptionScriptSelector()
+        self.ui.setupUi(self)
+        self.ui.button_box.accepted.connect(self.save_changes)
+        self.ui.button_box.rejected.connect(self.reject)
+        self.move_view = MoveableListView(self.ui.selected_scripts, self.ui.move_up, self.ui.move_down)
+        self.ui.add_script.clicked.connect(self.add_script)
+        self.ui.remove_script.clicked.connect(self.remove_script)
+        self.ui.selected_scripts.currentRowChanged.connect(self.selected_script_changed)
+        self.ui.weighting_selector.valueChanged.connect(self.weighting_changed)
+        self.load()
+
+    @staticmethod
+    def make_label(script_id, script_weighting):
+        return "{script} ({weighting}%)".format(
+            script=_(SCRIPTS[script_id]),
+            weighting=script_weighting,
+        )
+
+    def load(self):
+        self.ui.selected_scripts.clear()
+        for script in self.parent().current_scripts:
+            label = self.make_label(script_id=script[0], script_weighting=script[1])
+            item = QtWidgets.QListWidgetItem(label)
+            item.setData(QtCore.Qt.UserRole, script)
+            self.ui.selected_scripts.addItem(item)
+        if self.ui.selected_scripts.count() > 0:
+            self.ui.selected_scripts.setCurrentRow(0)
+        self.set_weighting_selector()
+
+        self.ui.available_scripts.clear()
+        for script_id in SCRIPTS:
```

Comment:
I think it would be great to have the scripts sorted by localized label, something like produced by this:

```
for script_id, label in sorted([(k, _(v)) for k, v in SCRIPTS.items()], key=lambda i: i[1]):
    ...
```